### PR TITLE
Refactor api-page.php file

### PR DESCRIPTION
**Pull request by JesusValera**
## 📚 Description

- Remove unused/deprecated imports from `build/api-page.php`
- Generate the `documentation/api.md` with missing commands

### CODE REVIEW

File: content/documentation/api.md [hunk 1]
```diff
@@ -26,6 +26,10 @@ numbers. If `xs` is empty, return 1.
 ```
 Return `a` to the power of `x`.
 
+## `*compile-mode*`
+
+
+
```
Comment (Chemaclass) [inline content/documentation/api.md:32]: There is no doc for this? 🤔 I will double check that, maybe for the next version we should add some information about this.

**Review by Chemaclass — APPROVED** [review]

File: content/documentation/api.md (hunk 1 above)
Comment (Chemaclass) [reply content/documentation/api.md:32]: Screenshot 2021-12-31 at 13 04 30

Interesting 🧐 the doc is there in the code, but when I run `(doc *compile-mode*)` there is no doc?
Any ideas about it, @jenshaase?

Comment (jenshaase) [reply content/documentation/api.md:32]: I think I know the problem: `doc` looks in `$GLOBALS` for the documentation. But not in GlobalEnvironment class. I will not fix it now. There needs to be a bigger refactoring of GlobalEnvironment and $GLOBALS in future.

**Comment by jenshaase** [comment]
By the way: I automatically build the API page during deployment. So there is no more need to regenerate it manually.

**Comment by Chemaclass** [comment]
Then we could remove this file and add it to .gitignore? Otherwise it will be out of sync for no reason. What do you think?

**Pull request merged**